server/src/main/java/au/org/aodn/ogcapi/server/core/model/CitationModel.java

```diff
@@ -0,0 +1,36 @@
+package au.org.aodn.ogcapi.server.core.model;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+
+@Data
```

Comment:
Please make it as generic so you can use it somewhere else, and refactor other objectMapper if you plan to do so, also do not new an objectmapper, you can do autowrite, i believe there is a util class ?

Reply:
Thank you. Test cases will added soon. 

On the other hand, i didn't find a util class. I will create one